src/controllers/loginController.js

```diff
@@ -0,0 +1,30 @@
+const loginService = require('../services/login');
+
+const loginController = function loginController(req, res) {
+  if (req.body.username && req.body.password) {
+    loginService.authorizeUser(JSON.stringify(req.body))
+      .then((data) => {
+        if (data === false) {
+          const errorMessage = [{
+            message: 'Incorrect Username and/or Password!',
+          }];
+          res.status(404).send(errorMessage);
```

Comment:
I am not sure if this is how the frontend expects it, check up on that. Also not only here but at the other error responses: Creating const errorMessage variable for every case like this is not a good practice in my opinion. This would look better for me but please ask the mentor: 
if (data === false) {
          res.status(404).send(errorMessage = {message: 'Incorrect Username and/or Password!'});